tests/functional/test_loaded_model.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import grpc
+import pytest
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+import tritonclient.grpc as grpcclient
+from model_config import MODEL_CONFIG
+PORT=8111
+
+
+@pytest.mark.parametrize("model", MODEL_CONFIG.keys())
+def test_inference(model):
+    triton_client = grpcclient.InferenceServerClient(
+        url=f"localhost:{PORT}",
+        verbose=False)
```

Comment:
this can be enclosed in a separate function, e.g. create_triton_client() to use it later in various tests